assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/AssignDatesButton.tsx

```diff
@@ -20,17 +20,16 @@ const AssignDatesButton: React.FC<EntityListItemProps<Ticket>> = React.memo(({ e
 		entity: 'tickets',
 		entityId: entity.id,
 	});
-	const count = relatedDatetimes.length;
 
-	const relatedDatetimeDbIds = relatedDatetimes.map(({ dbId }) => dbId);
+	const count = relatedDatetimes.length;
 
 	const title = count
-		? `${__('Related Dates:')} ${relatedDatetimeDbIds.join(', ')}`
+		? `${__('Number of related dates:')} ${count}`
```

Comment:
saem here:

```suggestion
		? __('Number of related dates:')
```

Reply:
applied, but w/o colon to make it consistent to changes in the other file